### PR TITLE
Added Nashik in locations section

### PR DESCRIPTION
**Pull request by aniketsingh98571**

### CODE REVIEW

File: locations.txt [hunk 1]
```diff
@@ -149,3 +149,4 @@ Visakhapatnam, India
 Victor Onyenma, Nigeria
 Jamshepur,India
 Burdwan,India
+Nashik India
```
Comment (sampoder) [inline locations.txt:152]: Nashik, India

Comment (aniketsingh98571) [reply locations.txt:152]: Great

Comment (sampoder) [reply locations.txt:152]: I think you need to fix t

Comment (sampoder) [reply locations.txt]: Fixed for you mate!

**Comment by TheShubham99** [comment]
Nashik is already present. Please make sure there are no redundant cities before merging :)

**Pull request merged**